test/boost/comparable_bytes_test.cc

```diff
@@ -16,6 +16,8 @@
 #include "utils/assert.hh"
```

Comment:
Is this legacy comparison order also binding to us?
Does Cassandra or we use it in already existing sstable formats?